sensors/hall_effect_sensor/rpm/hall_effect_sensor.ino

```diff
@@ -0,0 +1,97 @@
+#include "hall_effect_sensor.h"
+
+
+HallEffectSensor::HallEffectSensor(int pin = 2) : Sensor(pin)
+{
+    init_sensor(pin);
+}
+
+void Sensor::init_sensor(const int pin)
+{
+    Serial.begin(9600);
```

Comment:
This should be taken care of by the user i believe. Not needed in the class definition